### DISCUSSION TITLE
Health Fraction as a Library

### DISCUSSION THREAD

**Discussion by oOve** [mclemente/healthEstimate#89] [Ideas]
Hei, 
author of [swarm](https://github.com/oOve/swarm), [destructible tokens](https://github.com/oOve/destructibles), [projectiles](https://github.com/oOve/projectiles) and [follow me](https://github.com/oOve/FollowMe) here, all of which could benefit from accessing health of a given token as a fraction of its max. I see you already have such a method. Would it be possible to extract some of this functionality as a library that can be used by multiple modules?

**Comment by mclemente** [comment]
I could do that. Would you need any other functions besides getFraction?

**Reply by oOve** [reply]
Personally, only the fraction, though an api to heal x, or damage x would probably be useful as well.
